src/pages/specs/Search.spec.js

```diff
@@ -0,0 +1,18 @@
+import { create } from 'react-test-renderer';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
import { create } from "react-test-renderer";
```